### ISSUE TITLE
This pref can be replaced by `security.certerror.hideAddException`

### ISSUE THREAD

**Issue by mvk-foo** [arkenfox/user.js#1602]
This pref can be replaced by `security.certerror.hideAddException`
See https://bugzilla.mozilla.org/show_bug.cgi?id=1450761
Test https://expired.badssl.com

**This is especially true for users from Russia, where a self-signed Sberbank certificate is being implemented.**
https://bugzilla.mozilla.org/show_bug.cgi?id=1758773

_Originally posted by @mvk-foo in https://github.com/arkenfox/user.js/issues/1560#issuecomment-1345280844_

**Comment by dartraiden** [comment]
As a user from Russia, I am against it.

Periodically there are sites with expired or self-issued certificates. Sites hosted outside the Russian Federation. For example, the largest Russian-language computer forum uses manual renewal of certificates (they refuse to use scripts for this), so every few months the certificate expires and became invalid for 2-3 days.

The user must decide whether to ignore the invalid certificate or not.

**Comment by dartraiden** [comment]
> Sberbank

And this is also not a mitm. Some Russian companies cannot obtain a certificate because certification centers outside of Russia refuse to work with them.

The only way out was the creation of a Russian certification center. The alternative - using unencrypted http - is much worse.

**Comment by rusty-snake** [comment]
It's an two step procedure in Firefox. You have to click two grey buttons next to a recommened blue button.

Arkenfox makes it a one step procedure.
https://github.com/arkenfox/user.js/blob/7135907b2fe13fa55eb8ebf162603037f83e353c/user.js#L523

**Comment by mvk-foo** [comment]
> It _might_ be a useful pref to add to the harden section, but I would be loathe to add it as active.

Yes, in conjunction with https://blog.torproject.org/life-without-ca it can be added to the harden section.

**Comment by fxbrit** [comment]
I think we can ignore this pref: what's the value in hiding the UI button?

there are already a ton of warnings on that same page, it seems overkill. this pref IMO only makes sense in a corporate environment where you want to prevent employees from clicking the "Accept the risk" button because you don't want to trust them.
